editors/code/package.json

```diff
@@ -1494,4 +1507,4 @@
             ]
         }
     }
-}
+}
```

Comment:
Nit: let's keep the final newline.

Reply:
Yes sorry, it was automatically formatted by my code editor